downstream.stages

```diff
@@ -5,6 +5,6 @@ stage('Downstream Build') {
         println "Reading file ${REPOSITORY_LIST_FILE}"
         def file = readFile REPOSITORY_LIST_FILE
         def projectCollection = file.readLines()
-        treebuild.downstreamBuild(projectCollection, "${SETTINGS_XML_ID}", '-e -nsu -fae -Pbusiness-central,wildfly,sourcemaps,no-showcase clean install -Dfull=true -Dcontainer=wildfly -Dcontainer.profile=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -Dmaven.test.redirectTestOutputToFile=true -Dgwt.compiler.localWorkers=1')
+        treebuild.downstreamBuild(projectCollection, "${SETTINGS_XML_ID}", '-e -nsu -Pbusiness-central,wildfly,sourcemaps,no-showcase clean install -Dfull=true -Dcontainer=wildfly -Dcontainer.profile=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -Dmaven.test.redirectTestOutputToFile=true -Dgwt.compiler.localWorkers=1')
```

Comment:
where is this `fae` kept ?

Reply:
it is always specified here https://github.com/kiegroup/jenkins-pipeline-shared-libraries/blob/43f635dec3d19c136c8ffc1e4a457084cd61b923/vars/maven.groovy#L11